### DISCUSSION TITLE
Can't find missing images in icons directory

### DISCUSSION THREAD

**Discussion by Paulml1982** [navit-gps/navit#1238] [General]
Hi,

I am using navit 0.5.5 with Raspberry PI 4 Bullseye OS and i am trying to fix some errors navit throws. One of the errors is that it cannot find some xpm files from the icons directory. I was looking for those files in the trunk and other branches but it looks like they just don't exists. I use one of the default layout files that can be found in the repository. One error i get is the following:

error:navit:graphics_images_new_scaled_rotated: No image for '/usr/share/navit/icons/shop_handg.xpm'

Am i missing files or are there references to files that just don't exists anymore?

**Comment by lains** [comment]
Hello.
This is probably a missing icon, however, it is indeed referenced by several layout .xml files.
To fix this, this icon needs to be designed and added to the existing repository.
I guess the errors shows in the log, on the user interface, but the display is not broken and the application does not crash either.
